src/System.Windows.Forms/src/System/Windows/Forms/ContainerControl.cs

```diff
@@ -81,6 +81,11 @@ public class ContainerControl : ScrollableControl, IContainerControl
 
         private const string FontMeasureString = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
 
+        /// <summary>
+        /// True if the container needs scaling as a result of <see cref="DpiChangedEventHandler"/>, irrespective of Font status (explicit set or inherited Font).
+        /// </summary>
+        internal bool _needsDpiChangeScaling;
```

Comment:
1. What does this mean: "_...as a result of \<see cref="DpiChangedEventHandler"/\>_"?
2. Do not expose fields
3. Suggest different naming to semi-align with existing `IsScalingRequired` names.


```suggestion
        /// <summary>
        /// Gets or sets whether the container needs to be rescaled, irrespective whether the font was inherited or set explicitly.
        /// </summary>
        internal bool IsDpiChangeScalingRequired { get; set; }
```